lib/MetaCPAN/Document/File/Set.pm

```diff
@@ -475,5 +477,63 @@ sub autocomplete {
     )->sort( [ '_score', 'documentation' ] );
 }
 
+# this method will replace 'sub autocomplete' after the
+# mapping + data is fully deployed.
+# -- Mickey
+sub autocomplete_new {
```

Comment:
Can we find a name that doesn't include a `_new` suffix?  I see these at $work and they're baffling to everybody who wasn't around when they were created.